intelmq/bots/collectors/mail/README.md

```diff
@@ -0,0 +1,25 @@
+# Mail collector bots
```

Comment:
Remove all README.md files on this pull request and add the documentation text to the following location:
https://github.com/certtools/intelmq/blob/master/docs/Bots.md#collectors